raiden-dapp/src/locales/en.json

```diff
@@ -263,6 +263,7 @@
       "empty": "Enter an address or ENS name",
       "ens-resolve-failed": "Please enter a resolvable ENS name",
       "invalid-excluded-address": "Please enter a different address or ENS name. You can't use your own address or the address of the token network.",
+      "invalid-restricted-address": "Please enter a different address or ENS name. The used address is not allowed to be used here.",
```

Comment:
I think I commented on this in the last PR, so I guess it will be changed here once you merge that one.

Reply:
You refer to [this comment](https://github.com/raiden-network/light-client/pull/2921#discussion_r701241354). No, this PR becomes before the other one. So I'll do it here.

Reply:
✔️